src/WSASoap.php

```diff
@@ -41,21 +41,25 @@
  * POSSIBILITY OF SUCH DAMAGE.
  *
  * @author    Robert Richards <rrichards@ctindustries.net>
- * @copyright 2007-2017 Robert Richards <rrichards@ctindustries.net>
+ * @copyright 2007-2018 Robert Richards <rrichards@ctindustries.net>
  * @license   http://www.opensource.org/licenses/bsd-license.php  BSD License
  *
  * @version   2.0.3-dev
  */
 class WSASoap
 {
-    const WSANS = 'http://schemas.xmlsoap.org/ws/2004/08/addressing';
```

Comment:
I would leave this constant named as it is to prevent any possible BC issues in the event someone is referencing it for some reason. Outside of that the rest looks fine

Reply:
As nothing happens to this PR, I've created a new one, including @robrichards ' suggested fix